ResearchKit/ActiveTasks/ORKAudioContentView.m

```diff
@@ -47,7 +47,7 @@ @interface ORKAudioGraphView : UIView
 @property (nonatomic, strong) UIColor *keyColor;
 @property (nonatomic, strong) UIColor *alertColor;
 
-@property (nonatomic, copy) NSArray *values;
+@property (nonatomic, strong) NSArray *values;
```

Comment:
I believe his comment referred to a different place, but I could be wrong. This being copy is intended and is part of the original codebase.